lib/ansible/modules/network/nxos/nxos_linkagg.py

```diff
@@ -0,0 +1,401 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2017, Ansible by Red Hat, inc
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'network'}
+
+DOCUMENTATION = """
+---
+module: nxos_portchannel
```

Comment:
`module: nxos_linkagg`
Is what's causing the
`lib/ansible/modules/network/nxos/nxos_linkagg.py:0:0: E305 DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'nxos_portchannel'`